src/Compute/Compute/Generated/DedicatedHost/DedicatedHostGetMethod.cs

```diff
@@ -48,8 +48,8 @@ public override void ExecuteCmdlet()
                 {
                     case "ResourceIdParameter":
                         resourceGroupName = GetResourceGroupName(this.ResourceId);
-                        hostGroupName = GetResourceName(this.ResourceId, "Microsoft.Compute/hosts", "hosts");
```

Comment:
Where is the recorded test to show this works?

Reply:
Explain why this change makes it work in the PR description.

Reply:
I manually debug tested it, but will put up a revision with actual record testing too.

Reply:
Added recorded tests too

Reply:
Added reasoning for working of the change.